lib/player.ex

```diff
@@ -7,8 +7,8 @@ defmodule Player do
     "X"
   end
 
-  def move(board, space, mark) do
-    Board.make_mark(board, space, mark)
+  def move(board, space) do
+    Board.make_mark(board, space, mark())
```

Comment:
I'm curious about why `mark` is a function, and not just a variable. Why did you write it this way?